ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -29,6 +30,6 @@
     version: "{{ version }}"
 
 - name: install the catalog from the catalog location
-  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ cli.path }} chdir="{{ catalog_location }}/packages"
+  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} {{ whisk_api_host }} chdir="{{ catalog_location }}/packages"
```

Comment:
@narinearraul the CLI already knows about the `APIHOST`. At a quick glance on `installCatalog.sh` I couldn't tell easily which script would require direct HTTP API Access vs `wsk` CLI. Do you happen to know this ?